### ISSUE TITLE
Sync seems to do nothing

### ISSUE THREAD

**Issue by Arthur-Milchior** [ankitects/anki#2173]
Running on bf34ac77672b615ea1def2a1088ac0ced0c4fc20
When I click on "sync", sometime I do not see anything. 
Maybe the sync was actually extremely quick, and so the progress bar did not ave time to appear.
Maybe anki decided not to do too sync one after the other to save on bandwidth. In any case, the total absence of any feedback when a button is pressed (appart from the change in color while the mouse is is pressed) is not a good user experience. I'd suggest either a minimal duration for the progress bar or a tooltip when synced

**Comment by dae** [comment]
The rationale here is that the sync button will be a different color if there are pending changes, and will change color after syncing.

**Comment by Arthur-Milchior** [comment]
Oh, great. So I guess I can close it if it's not a bug, but just a partially implemented feature

**Comment by dae** [comment]
What's partially implemented? The color changing should already be happening.

**Comment by Arthur-Milchior** [comment]
So I filled #2178, which may explain why I was not seeing the color change. (I.e. because it only change once I go back to deck picker, not during reviews)

In any case, right now, it was really not clear to me when I pressed "sync" that the click had actually any effect. That either it checked whether there is anything to download from ankiweb, or that it rejected because it made an educated guess that it's useless to sync again 20 seconds after last sync (which seems reasonable). And for what I know about UI, it remains generally a bad idea to give 0 feedback after a button pressed (unless of course the button was disabled in the first place. Which may make sense)

Relatedly, in dark mode, the difference between "Sync" in blue and in white is very subtil. It's really not clear, until you know it, that this blue transmit any information about the state of the system. Actually, when you open the preference, the word basic is in a relatively similar blue. 
![2022-11-03-055812_940x510_scrot](https://user-images.githubusercontent.com/357361/199650483-8d7d11ad-35ed-4309-af09-541709072143.png)

As a user, I really feel like the blue of "Sync" indicates that there is a difference of focus, that maybe it would be a default action, or a selected component. But it's really not intuitive that this colors transmits a message on the state of the system. But maybe I'm just too used to the labels that AnkiDroid has to indicate when a partial/full sync is needed.
